app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "cinema"
+
+    def create(self, first_name_: str, last_name_: str) -> None:
```

Comment:
Why do you add `_` to the end of variable names?

Reply:
Danylo shows such approach in video lectures that`s why I wrote in this way.

Reply:
okay, I will ask him)

Reply:
I rewatched video. The reason was reserved name, but I missed this mention. Shame on me )